src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.ts

```diff
@@ -25,6 +26,7 @@ export class FssSimplifiedSearchComponent implements OnInit {
 
   getSimplifiedSearchResult(){
     if (this.fieldValue.trim() !== "") {
+      var filterExpression=this.fssSearchFilterService.getFilterExpressionForSimplifiedSearch(this.fieldValue.trim());     
```

Comment:
Please use `let` or `const` here
```suggestion
      let filterExpression=this.fssSearchFilterService.getFilterExpressionForSimplifiedSearch(this.fieldValue.trim());     
```

Reply:
resolved